UI/window-basic-main-outputs.cpp

```diff
@@ -698,6 +719,19 @@ void SimpleOutput::UpdateRecordingSettings_amd_cqp(int cqp)
 	obs_encoder_update(videoRecording, settings);
 }
 
+#ifdef ENABLE_HEVC
+void SimpleOutput::UpdateRecordingSettings_nvenc_hevc(int cqp)
```

Comment:
Move this function up to be under UpdateRecordingSettings_nvenc

Reply:
Done.